### DISCUSSION TITLE
Sharepoint theme values between targeted and standard release tenants different/missing

### DISCUSSION THREAD

**Discussion by ferrarirosso** [microsoft-search/pnp-modern-search#2297] [Q&A] [answered]
Since a couple of days, we have various theme properties missing on our targeted release tenants.

The properties between the two tenants looks quite different and we have now some templating which doesn't render correctly anymore, example property PrimaryText is missing in the targeted release tenants.

1. I suppose this is related to the tenant "mode" since we have this issue newly on 2 targeted releases tenant but not on the standard one.
2. Is this a known change in the sharepoint theming ? I did a quick search in the office 365 roadmap and did not found anything related.

Thanks & kind regards

### Excerpt from one of our targeted release tenant (debug output from the pnp modern search)
  "theme": {
    "palette": {
      "themeDarker": "#791313",
      "themeDark": "#a51a1a",
      "themeDarkAlt": "#c31f1f",
      "themePrimary": "#da2323",
      "themeSecondary": "#dd3a3a",
      "themeTertiary": "#e87373",
      "themeLight": "#f4b6b6",
      "themeLighter": "#f9d7d7",
      "themeLighterAlt": "#d4edf2",
      "black": "#000000",
      "blackTranslucent40": "rgba(0,0,0,.4)",
      "neutralDark": "#141414",
      "neutralPrimary": "#242424",
      "neutralPrimaryAlt": "#424242",
      "neutralSecondary": "#616161",
      "neutralSecondaryAlt": "#8A8A8A",
      "neutralTertiary": "#9E9E9E",
      "neutralTertiaryAlt": "#C7C7C7",
      "neutralQuaternary": "#D1D1D1",
      "neutralQuaternaryAlt": "#E0E0E0",
      "neutralLight": "#EBEBEB",
      "neutralLighter": "#F0F0F0",
      "neutralLighterAlt": "#FAFAFA",
      "accent": "#da2323",
      "white": "#FFFFFF",
      "whiteTranslucent40": "rgba(255,255,255,.4)",
      "yellowDark": "#d29200",
      "yellow": "#ffb900",
      "yellowLight": "#fff100",
      "orange": "#d83b01",
      "orangeLight": "#ea4300",
      "orangeLighter": "#ff8c00",
      "redDark": "#a4262c",
      "red": "#e81123",
      "magentaDark": "#5c005c",
      "magenta": "#b4009e",
      "magentaLight": "#e3008c",
      "purpleDark": "#32145a",
      "purple": "#5c2d91",
      "purpleLight": "#b4a0ff",
      "blueDark": "#002050",
      "blueMid": "#00188f",
      "blue": "#0078d4",
      "blueLight": "#00bcf2",
      "tealDark": "#004b50",
      "teal": "#008272",
      "tealLight": "#00b294",
      "greenDark": "#004b1c",
      "green": "#107c10",
      "greenLight": "#bad80a",
      "grey96": "#f5f5f5",
      "grey74": "#bdbdbd"
    },

### Excerpt from our standard tenant (debug output from the pnp modern search)
  "theme": {
    "palette": {
      "themeDarker": "#791313",
      "themeDark": "#a51a1a",
      "themeDarkAlt": "#c31f1f",
      "themePrimary": "#da2323",
      "themeSecondary": "#dd3a3a",
      "themeTertiary": "#e87373",
      "themeLight": "#f4b6b6",
      "themeLighter": "#f9d7d7",
      "themeLighterAlt": "#d4edf2",
      "black": "#000000",
      "blackTranslucent40": "rgba(0,0,0,.4)",
      "neutralDark": "#201f1e",
      "neutralPrimary": "#323130",
      "neutralPrimaryAlt": "#3b3a39",
      "neutralSecondary": "#605e5c",
      "neutralSecondaryAlt": "#8a8886",
      "neutralTertiary": "#a19f9d",
      "neutralTertiaryAlt": "#c8c6c4",
      "neutralQuaternary": "#d0d0d0",
      "neutralQuaternaryAlt": "#e1dfdd",
      "neutralLight": "#edebe9",
      "neutralLighter": "#f3f2f1",
      "neutralLighterAlt": "#faf9f8",
      "accent": "#da2323",
      "white": "#ffffff",
      "whiteTranslucent40": "rgba(255, 255, 255, 0.40)",
      "yellowDark": "#d29200",
      "yellow": "#ffb900",
      "yellowLight": "#fff100",
      "orange": "#d83b01",
      "orangeLight": "#ea4300",
      "orangeLighter": "#ff8c00",
      "redDark": "#a4262c",
      "red": "#e81123",
      "magentaDark": "#5c005c",
      "magenta": "#b4009e",
      "magentaLight": "#e3008c",
      "purpleDark": "#32145a",
      "purple": "#5c2d91",
      "purpleLight": "#b4a0ff",
      "blueDark": "#002050",
      "blueMid": "#00188f",
      "blue": "#0078d4",
      "blueLight": "#00bcf2",
      "tealDark": "#004b50",
      "teal": "#008272",
      "tealLight": "#00b294",
      "greenDark": "#004b1c",
      "green": "#107c10",
      "greenLight": "#bad80a",
      "HyperlinkActive": "#791313",
      "CommandLinksPressed": "#791313",
      "NavigationPressed": "#791313",
      "EmphasisHoverBorder": "#791313",
      "TopBarPressedText": "#791313",
      "HeaderNavigationPressedText": "#791313",
      "Hyperlinkfollowed": "#791313",
      "EmphasisHoverBackground": "#a51a1a",
      "EmphasisBorder": "#a51a1a",
      "AccentText": "#da2323",
      "CommandLinksHover": "#da2323",
      "RowAccent": "#da2323",
      "NavigationAccent": "#da2323",
      "NavigationHover": "#da2323",
      "EmphasisBackground": "#da2323",
      "HeaderNavigationHoverText": "#da2323",
      "HeaderNavigationSelectedText": "#da2323",
      "SuiteBarBackground": "#da2323",
      "Hyperlink": "#da2323",
      "ContentAccent1": "#da2323",
      "AccentLines": "#dd3a3a",
      "HeaderAccentLines": "#dd3a3a",
      "ButtonPressedBorder": "#dd3a3a",
      "SuiteBarHoverBackground": "#e87373",
      "StrongLines": "#f4b6b6",
      "HeaderStrongLines": "#f4b6b6",
      "SuiteBarHoverText": "#f4b6b6",
      "ButtonPressedBackground": "#f4b6b6",
      "ButtonHoverBorder": "#f4b6b6",
      "ButtonHoverBackground": "#f9d7d7",
      "SelectionBackground": "rgba(244, 182, 182, 0.50)",
      "HoverBackground": "rgba(249, 215, 215, 0.50)",
      "NavigationHoverBackground": "rgba(249, 215, 215, 0.50)",
      "PageBackground": "#ffffff",
      "EmphasisText": "#ffffff",
      "SuiteBarText": "#ffffff",
      "TileText": "#ffffff",
      "BackgroundOverlay": "rgba(255, 255, 255, 0.85)",
      "HeaderBackground": "rgba(255, 255, 255, 0.85)",
      "FooterBackground": "rgba(255, 255, 255, 0.85)",
      "DisabledBackground": "#faf9f8",
      "HeaderDisabledBackground": "#faf9f8",
      "ButtonBackground": "#faf9f8",
      "ButtonDisabledBackground": "#faf9f8",
      "SubtleEmphasisBackground": "#f3f2f1",
      "DialogBorder": "#f3f2f1",
      "NavigationSelectedBackground": "rgba(237, 235, 233, 0.78)",
      "TopBarBackground": "rgba(237, 235, 233, 0.78)",
      "DisabledLines": "#edebe9",
      "HeaderDisabledLines": "#edebe9",
      "ButtonDisabledBorder": "#edebe9",
      "SuiteBarDisabledText": "#edebe9",
      "SubtleLines": "#c8c6c4",
      "HeaderSubtleLines": "#c8c6c4",
      "ButtonGlyphDisabled": "#c8c6c4",
      "DisabledText": "#a19f9d",
      "CommandLinksDisabled": "#a19f9d",
      "HeaderDisableText": "#a19f9d",
      "ButtonDisabledText": "#a19f9d",
      "Lines": "#a19f9d",
      "HeaderLines": "#a19f9d",
      "ButtonBorder": "#a19f9d",
      "CommandLinks": "#605e5c",
      "Navigation": "#605e5c",
      "SubtleEmphasisText": "#605e5c",
      "TopBarText": "#605e5c",
      "HeaderNavigationText": "#605e5c",
      "ButtonGlyph": "#605e5c",
      "BodyText": "#323130",
      "WebPartHeading": "#323130",
      "HeaderText": "#323130",
      "ButtonText": "#323130",
      "ButtonGlyphActive": "#323130",
      "TopBarHoverText": "#323130",
      "StrongBodyText": "#201f1e",
      "SiteTitle": "#201f1e",
      "CommandLinksSecondary": "#201f1e",
      "SubtleEmphasisCommandLinks": "#201f1e",
      "HeaderSiteTitle": "#201f1e",
      "TileBackgroundOverlay": "rgba(0, 0, 0, 0.50)",
      "ContentAccent2": "#00485b",
      "ContentAccent3": "#288054",
      "ContentAccent4": "#767956",
      "ContentAccent5": "#ed0033",
      "ContentAccent6": "#682a7a",
      "backgroundOverlay": "rgba(255, 255, 255, 0.85)",
      "themeAccent": "#da2323",
      "themeAccentTranslucent10": "rgba(218, 35, 35, 0.10)",
      "primaryBackground": "#ffffff",
      "primaryText": "#333333",
      "suiteBarBackground": "#da2323",
      "suiteBarText": "#ffffff",
      "suiteBarDisabledText": "#edebe9",
      "topBarBackground": "rgba(237, 235, 233, 0.78)",
      "topBarText": "#605e5c",
      "topBarHoverText": "#323130",
      "dialogBorder": "#f3f2f1"
    },

**Comment by FranckyC** [comment, marked as answer]
Hi @ferrarirosso, this is likely not a Web Part issue of course as we simply output what is available from the SPFx context. As you stated _"I suppose this is related to the tenant "mode" since we have this issue newly on 2 targeted releases tenant but not on the standard one"_,  I think you have your explanation already :D.Maybe you should ask your question here: https://github.com/SharePoint/sp-dev-docs. Thanks!

**Reply by ferrarirosso** [reply]
### Update
The problem is not (only ?) related to the tenant mode
We have some user having the properties and some users not having these properties.

**Reply by FranckyC** [reply]
That's dumb but could be related to browser cache. Can you test using private windows?

**Reply by ferrarirosso** [reply]
Yes we have deleted cache and use private windows, doesn't change, appears that some users are having the issue since a couple of days
Best workaround for us is to define the styling "hard coded" in the template and not use the missing properties from the SP Theme.
Thanks you for your assistance Frank !

**Reply by FranckyC** [reply]
Sure, unfortunately, nothing we can do here :/
